graphql/database/backgroundImages/BackgroundImageCategoryModel.js

```diff
@@ -0,0 +1,51 @@
+import BaseModel from '../base/BaseModel'
+import types from '../fieldTypes'
+import tableNames from '../tables'
+import { BACKGROUND_IMAGE_CATEGORY } from '../constants'
+
+/*
+ * @extends BaseModel
+ */
+class BackgroundImageCategory extends BaseModel {
+  static get name() {
+    return BACKGROUND_IMAGE_CATEGORY
+  }
+
+  static get hashKey() {
```

Comment:
Let's use a nanoid as the hash key.

Reply:
The name can be a GSI (primary lookup for now). As we automate background images, we'd shift over to ID-based lookup.